File: pset6/mario.py

height = 0
goodinput = False
while not goodinput:
    try:
        number = int(input('Enter a number: '))
        if number > 0 and number <= 23:
            goodinput = True
            height = number
        else:
            print("We need a number between 0 and 24 please!")
    except ValueError:
        print("that's not an integer. Try again.")
        

for i in range(height):
    space = height - i
    for j in range(space, 1, -1):
        print(" ", end="")
    
    for hash in range(0, i + 2):
        print("#", end="")
        
    print("")
        

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int height;

int main(void)
{
    do
    {
        printf("Height: ");
        height = GetInt();
    }
    while (height < 0 || height > 23);
    
    for(int i = 0; i < height; i++ )
    {
        for(int space = height - i; space > 1; space--)
        {
            printf(" ");    
        }
        for(int hash = 0; hash <= i + 1; hash++)
        {
            printf("#");
        }
        printf("\n");
    }
}